homeassistant/components/motionblinds_ble/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "domain": "motionblinds_ble",
+  "name": "MotionBlinds BLE",
+  "bluetooth": [
+    {
+      "local_name": "MOTION_****",
+      "connectable": true
+    }
+  ],
+  "codeowners": ["@LennP", "@jerrybboy"],
+  "config_flow": true,
+  "dependencies": ["bluetooth_adapters"],
+  "documentation": "https://www.home-assistant.io/integrations/MotionBlinds_BLE",
+  "homekit": {},
+  "iot_class": "assumed_state",
+  "issue_tracker": "https://github.com/LennP/HA-MotionBlinds_BLE/",
```

Comment:
This should not be specified for core integrations.

Reply:
Thanks, I'll remove them!

Reply:
Most core integrations I see do have keys `documentation` and `iot_class`. I did remove `ssd`, `zeroconf`, `homekit` and `issuetracker`.

Reply:
Oh sorry, I only meant to comment the `issue_tracker` key. `documentation` and `iot_class` should stay there of course. `homekit` could be removed since it's empty anyway. 👍